index.html

```diff
@@ -100,7 +100,7 @@ <h2>I'am a software developer</h2>
         </div>
 
         <div class="illustration-header">
-          <img src="/images/Illustration-header.png" alt="">
+          <img src="../images/Illustration-header.png" alt="">
```

Comment:
- [ ] Good job deploying the website with Github pages. Currently, the background images are not getting displayed, kindly fix your image paths to display them correctly. 🤝️
Please do the same thing for the mobile version too. 😎️

![Screenshot from 2023-01-23 12-28-18](https://user-images.githubusercontent.com/17243325/214007087-28c97ca5-caba-4a49-83e1-f8665ae81a5d.png)

![Screenshot from 2023-01-23 12-28-35](https://user-images.githubusercontent.com/17243325/214007172-0a35f1ae-0887-4a5e-9ac6-c7453079d309.png)